lib/linguist/languages.yml

```diff
@@ -2716,6 +2716,20 @@ Kaitai Struct:
   - ".ksy"
   tm_scope: source.yaml
   language_id: 818804755
+KakouneScript:
+  type: programming
+  color: "#6f8042"
+  tm_scope: source.kakscript
+  aliases:
+  - kak
+  - kakscript
+  extensions:
+  - ".kak"
+  filenames:
+  - kakrc
```

Comment:
Please add a sample `kakrc` file to this pull request and update the template to link to the source and state the license of the file.